builtin/providers/azurerm/resource_arm_eventhub_namespace.go

```diff
@@ -0,0 +1,210 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/arm/eventhub"
+	"github.com/hashicorp/terraform/helper/schema"
+	"net/http"
+)
+
+// Default Authorization Rule/Policy created by Azure, used to populate the
+// default connection strings and keys
+var eventHubNamespaceDefaultAuthorizationRule = "RootManageSharedAccessKey"
+
+func resourceArmEventHubNamespace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmEventHubNamespaceCreate,
+		Read:   resourceArmEventHubNamespaceRead,
+		Update: resourceArmEventHubNamespaceCreate,
+		Delete: resourceArmEventHubNamespaceDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
Added a test but it's failing - posting this here for later as a reminder of what needs fixing:

```
TF_ACC=1 go test ./builtin/providers/azurerm -v -run TestAccAzureRMEventHubNamespace_importBasic -timeout 120m
=== RUN   TestAccAzureRMEventHubNamespace_importBasic
--- FAIL: TestAccAzureRMEventHubNamespace_importBasic (371.43s)
	testing.go:265: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
		
		(map[string]string) {
		}
		
		
		(map[string]string) (len=2) {
		 (string) (len=8) "location": (string) (len=6) "westus",
		 (string) (len=19) "resource_group_name": (string) (len=28) "acctestRG-342930116917413830"
		}
		
FAIL
exit status 1
FAIL	github.com/hashicorp/terraform/builtin/providers/azurerm	371.449s
```